src/content/reference/react-dom/components/index.md

```diff
@@ -1,40 +1,38 @@
 ---
-title: "React DOM Components"
+title: "Komponen DOM React"
 ---
 
 <Intro>
-
-React supports all of the browser built-in [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) and [SVG](https://developer.mozilla.org/en-US/docs/Web/SVG/Element) components.
-
+React mendukung semua komponen [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) dan [SVG](https://developer.mozilla.org/en-US/docs/Web/SVG/Element) bawaan peramban.
 </Intro>
 
 ---
 
-## Common components {/*common-components*/}
+## Komponen-Komponen Umum {/*common-components*/}
 
-All of the built-in browser components support some props and events.
+Semua komponen bawaan *browser* mendukung beberapa *props* dan *events*.
```

Comment:
looking at the previous translation of "browser", I think this one should be "peramban" instead of "browser".
or do we have a convention for this?

```suggestion
Semua komponen bawaan peramban mendukung beberapa *props* dan *events*.
```

Reply:
@itsfaqih It's peramban in the [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium)

Reply:
Updated, changed to `peramban`